src/components/SearchPage/SearchPage.jsx

```diff
@@ -0,0 +1,94 @@
+import { Link } from "react-router-dom";
+import React from "react";
+
+import Footer from "../Footer/Footer";
+import logo from "../../assets/logo.png";
+import "./SearchPage.css";
+import { fetchPokemon } from "../../utils/PokeApi";
+
+function InfoPage() {
+  function handleEnter(event) {
```

Comment:
You don't need `handleEnter` because you should set a `submit` handler for the `form` tag

Submit handlers are added only to the `form` tags with the `submit` event rather than to submit buttons with the `click` event because it also automatically handles `Enter` presses